.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
   "editor.tabSize": 2,
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": "explicit",
-    "source.fixAll.stylelint": true
+    "source.fixAll.stylelint": "explicit"
```

Comment:
This kept changing on its own. I didn't look into it but it's still working correctly.